infra/azure/terraform/main.tf

```diff
@@ -9,8 +9,8 @@ terraform {
 
 terraform {
   backend "azurerm" {
-    resource_group_name  = "***********************"
-    storage_account_name = "YOUR STORAGE ACCOUNT"
+    resource_group_name  = "titus-rg"
+    storage_account_name = "titusonazure"
```

Comment:
should these be dynamic?

Reply:
> Since you removed the input.tfvars file, do we need to update the instructions or they were already correct and the file should have not been there in the first place?

I don't think the input file should be on git. There is a `sample-input.tfvars` which can be copy-pasted and customized with the information needed

Reply:
oh sorry, variables are not allowed into the `backend` section, so they must be hardwired

Reply:
anything that may likely be customized (and I believe these two are) should be in variables or anywhere it makes sense to allow users to customize them. let's avoid hardcoded values which we know shouldn't be hardcoded

Reply:
ok but even if they are not allowed, shall we hardcode them to values that we know are not portable?

Reply:
Actually storing terraform state in a storage account is up to the user, so we can comment this part and use the local storage (as we do in AWS)